src/coreclr/jit/codegencommon.cpp

```diff
@@ -1966,6 +1966,11 @@ void CodeGen::genEmitMachineCode()
     trackedStackPtrsContig = !compiler->opts.compDbgEnC;
 #endif
 
+    if (compiler->opts.disAsm)
+    {
+        printf("; BEGIN METHOD ANCHOR %s\n", compiler->eeGetMethodFullName(compiler->info.compMethodHnd));
```

Comment:
Since this also appears whenever people ask for disassembly perhaps we could change it to something more meaningful?

```suggestion
        printf("; BEGIN METHOD CODE %s\n", compiler->eeGetMethodFullName(compiler->info.compMethodHnd));
```
or maybe just

```suggestion
        printf("; BEGIN METHOD %s\n", compiler->eeGetMethodFullName(compiler->info.compMethodHnd));
```